pages/content/amp-dev/documentation/guides-and-tutorials/integrate/integrate-with-apps.md

```diff
@@ -110,7 +110,7 @@ The response body contains the AMP URL mapping in JSON format:
 
 ## Using AMP Caches
 
-An [AMP Cache]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/index.md', locale=doc.locale).url.path}}) is a
+An [AMP Cache]({{g.doc('/content/amp-dev/documentation/guides-and-tutorials/learn/amp-caches-and-cors/how_cached.md', locale=doc.locale).url.path}}) is a
```

Comment:
Sidenote: how comes that the URL does not reflect the actual title?

Reply:
You are able to define the URLs either per folder in their `_blueprint.yaml` or via `$path` in a files frontmatter. Now `how_cached.md` did latter and overwrote its natural path. Having it named `index.md` was coming from a quirk the sidebar previously needed.

Reply:
What I mean is, the page is named `How AMP pages are cached - why is the URL not `.../amp-caches-and-cors/how-amp-pages-are-cache/`?

Reply:
Oh I just went with the URL it had before: https://www.ampproject.org/docs/fundamentals/how_cached 😉 WDYT, update?

Reply:
Updated it to `how_amp_pages_are_cached` as most URLs share snake-case currently.